apps/frontend/src/pages/project/libs/components/project-details-menu/project-details-menu.tsx

```diff
@@ -30,11 +30,11 @@ const ProjectDetailsMenu = ({
 	);
 
 	const hasManageProjectAccessPermission = checkHasPermission(
-		[PermissionKey.MANAGE_USER_ACCESS],
+		[PermissionKey.MANAGE_USER_ACCESS, PermissionKey.MANAGE_ALL_PROJECTS],
 		userPermissions,
 	);
 	const hasEditProjectPermission = checkHasPermission(
-		[PermissionKey.MANAGE_ALL_PROJECTS],
+		[PermissionKey.MANAGE_ALL_PROJECTS, PermissionKey.MANAGE_USER_ACCESS],
```

Comment:
Users with MANAGE_USER_ACCESS shouldn't be able to edit projects  
```suggestion
		[PermissionKey.MANAGE_ALL_PROJECTS],
```

Reply:
Fixed, thanks for pointing that out